test/annexB/language/statements/try/catch-redeclared-for-of-var.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2019 Sony Interactive Entertainment Inc.
```

Comment:
Hmm, so the validator is complaining because I dropped the `All rights reserved.` here, which we [usually](https://github.com/WebKit/webkit/blob/master/Source/WebInspectorUI/UserInterface/Views/Layers3DContentView.js#L2) don't include in our copyright line.

I guess #851 is still outstanding, otherwise I could just drop it altogether. 😅

Reply:
I guess this is blocking the merge? Can this be addressed somehow?

Reply:
Hoping @leobalter or @littledan can advise. 🤞

Reply:
I'll apply your suggestion for now and others can object / accept when they get to it. 🙂